src/local_scheduler/local_scheduler_algorithm.cc

```diff
@@ -255,22 +263,19 @@ void remove_actor(SchedulingAlgorithmState *algorithm_state, ActorID actor_id) {
 
   /* Log some useful information about the actor that we're removing. */
   char id_string[ID_STRING_SIZE];
-  task_queue_entry *elt;
-  int count;
-  DL_COUNT(entry->task_queue, elt, count);
+  int64_t count = entry->task_queue->size();
   if (count > 0) {
     LOG_WARN("Removing actor with ID %s and %d remaining tasks.",
              ObjectID_to_string(actor_id, id_string, ID_STRING_SIZE), count);
```

Comment:
Using `printf` in C++ is going to be ugly no matter what you do. `%d` means `int`, so anything else is wrong... but won't necessarily give an error if the argument is passed in a register. `%ld` and `%lld%` mean `[long] long int`, but that works wrong on Windows. Similarly with `%u`, `%lu`, and `%llu` for `unsigned`. And all of these are wrong for `size_t` or `ptrdiff_t`, which is really what you're trying to print here.
I would suggest either leaving the warning as-is, or casting the argument with `static_cast<long long>` and using `%lld`, using http://stackoverflow.com/a/6400404 if you want to write ultra-correct C code, or use `std::stringstream` with `operator<<` if you want a C++ solution.